client/src/Component/BottomNav.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import { Footer } from 'react-materialize';
+
+class BottomNav extends Component {
+    render() {
+        return (
+            <div>
+                <Footer 
+                    copyrights="&copy; 2018 Zhenzhen You" 
+                    moreLinks={
+                        <a className="grey-text text-lighten-4 right" href="#!">More Links</a>
+                    }
+                    links={
+                        <ul>
+                        <li><a className="grey-text text-lighten-3" href="#!">Link 1</a></li>
```

Comment:
Is there a plan for these to link to something later?

Reply:
Yes, I will think about those links towards the end of this project...